addons/ui/functions/fnc_handleSpeedIndicator.sqf

```diff
@@ -24,5 +24,8 @@ private _isProne = _animState select [5, 3] isEqualTo "pne";
 
 private _icon = GVAR(speedIndicatorIconHash) getOrDefault [[_animSpeed, _isProne], ""];
 
-private _speedIndicator = uiNamespace getVariable [QGVAR(speedIndicator), controlNull];
-_speedIndicator ctrlSetText _icon;
+private _speedIndicatorContainer = uiNamespace getVariable [QGVAR(speedIndicatorContainer), controlNull];
+(_speedIndicatorContainer controlsGroupCtrl 10) ctrlSetText _icon;
+
+_speedIndicatorContainer ctrlSetFade 0;
+_speedIndicatorContainer ctrlCommit 0
```

Comment:
I see no need to "unfade" the indicator on every animation change. As this code runs "all the time" due to changing animations I would prefer for it to be as fast as possible. 

Also I would suggest to store the indicator itself in the uiNamspace so there will be no need for `controlsGroupCtrl `.